frontend/src/pages/Status.tsx

```diff
@@ -55,6 +55,32 @@ export function hasFinished(status?: NodePhase): boolean {
   }
 }
 
+export function statusToFadedColor(status?: NodePhase): string {
```

Comment:
How about `statusToNodeColor` or `statusToBgColor`?

Reply:
Should the colors then be `statusBgColors` ?